lib/engine/game/base.rb

```diff
@@ -1774,10 +1774,19 @@ def init_hexes(companies, corporations)
 
         reservations = Hash.new { |k, v| k[v] = [] }
         reservation_corporations.each do |c|
-          reservations[c.coordinates] << {
-            entity: c,
-            city: c.city,
-          }
+          if c.coordinates.is_a?(Array)
+            c.coordinates.each_with_index do |coord, idx|
```

Comment:
how hard is it to just change coordinates to always be an array?

Reply:
It would mean either
1. editing the the config files for all other games, or
2. adding the conversion to game/base when the CORPORATIONS variable is set from data['corporations'] (and for MINORS too).

However, I'd also have to comb through the code base to see where *.coordinates is used and modify them.

Reply:
That being, said, I can simplify this to not use an if.